app/assets/stylesheets/content/_work_packages_table.sass

```diff
@@ -25,41 +25,3 @@
 //
 // See doc/COPYRIGHT.rdoc for more details.
 //++
-
-@import 'table'
```

Comment:
It's because I know more than you do. 

@HDinger does alter the behaviour (min-width) of the columns again in https://github.com/HDinger/openproject/blob/feature/21231-apply-table-styling/app/assets/stylesheets/content/_work_packages_table.sass#L41

I therefore didn't want her to have to recreate the file again.